core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -562,10 +562,10 @@ $(document).ready(function () {
 
                 $("#showAdditionalMetrics").append(
                     "<div><a id='additionalMetrics'>" +
-                    "<span class='expand-input-rate-arrow arrow-closed' id='arrowtoggle-optional-metrics'></span>" +
+                    "<span class='expand-input-rate-arrow arrow-closed' id='arrowtoggle-optional-metrics'></span> " +
```

Comment:
Nit: this line seems only adding one space, we can avoid such changes.

Reply:
Agreed this is a little lame.  I'll remove the space and use styling instead.

Reply:
The space was removed.  It was originally added to make the spacing arrow spacing consistent between the executor page and the other pages since the other pages rely on whitespace for the gap between the arrow and the expand / collapse text.  Fixing all other occurences would require many more code changes so I just reverted the space for now.